### ISSUE TITLE
Possible to specify executable for subcommands?

### ISSUE THREAD

**Issue by jd20** [tj/commander.js#826]
I have a set of subcommands, structured like this:

```
cli.js
cli-search.js
cli.parse.js
```

And in my package.json, I have `"bin": { "mytool": "bin/cli.js" }`. When I run subcommands locally, using `node bin/cli.js`, all is fine. But when I install my project globally and try to run `mytool search` from the command line, it doesn't work, because it's looking for `mytool-search.js`.

Is it possible to override the default behavior of using `argv[1]`? I want to avoid having to rename everything to `mytool-search.js`, etc...

**Comment by jd20** [comment]
Hi, I created #830 to provide a fix for this problem. It does the common sense thing of checking what the symlink in bin resolved to, and checking for that, if it would otherwise fail to find the subcommand's script.

**Comment by pavelloz** [comment]
I dont know if it helps, but i always define executable as relative path and it always works, ie.

```
{
  "mytool": "./bin/cli.js"
}
```

**Comment by shadowspawn** [comment]
Pull Request being considered for v3 is #999

**Comment by shadowspawn** [comment]
This issue will be resolved when v3.0.0 is released. Available now as a prerelease. See #1001

**Comment by shadowspawn** [comment]
`.command` now supports the `executableFile` option (thanks to @abetomo). 
Shipped in v3: https://github.com/tj/commander.js/releases/tag/v3.0.0
